app/assets/javascripts/application.js

```diff
@@ -74,9 +74,9 @@ function openPopup (url, name, top, left) {
     })
 
     // Empty the error message list and repopulate with correct ordered list
-    $('.govuk-error-summary__list').empty()
+    $('.govuk-error-summary__list:not(.app-fixed-order)').empty()
```

Comment:
Special case of error page where we don't want to do all this error reordering logic